src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -2,79 +2,79 @@
-import seedu.address.logic.commands.EditCommand.EditPersonDescriptor;
+import seedu.address.logic.commands.EditCommand.EditModuleDescriptor;
 import seedu.address.model.GradPad;
 import seedu.address.model.Model;
 import seedu.address.model.ModelManager;
 import seedu.address.model.UserPrefs;
-import seedu.address.model.person.Person;
-import seedu.address.testutil.EditPersonDescriptorBuilder;
-import seedu.address.testutil.PersonBuilder;
+import seedu.address.model.module.Module;
+import seedu.address.testutil.EditModuleDescriptorBuilder;
+import seedu.address.testutil.ModuleBuilder;
 
 /**
  * Contains integration tests (interaction with the Model, UndoCommand and RedoCommand) and unit tests for EditCommand.
  */
 public class EditCommandTest {
 
-    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+    private Model model = new ModelManager(getTypicalGradPad(), new UserPrefs());
 
     @Test
     public void execute_allFieldsSpecifiedUnfilteredList_success() {
-        Person editedPerson = new PersonBuilder().build();
-        EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder(editedPerson).build();
-        EditCommand editCommand = new EditCommand(INDEX_FIRST_PERSON, descriptor);
+        Module editedModule = new ModuleBuilder().build();
+        EditModuleDescriptor descriptor = new EditModuleDescriptorBuilder(editedModule).build();
+        EditCommand editCommand = new EditCommand(INDEX_FIRST_MODULE, descriptor);
 
-        String expectedMessage = String.format(EditCommand.MESSAGE_EDIT_PERSON_SUCCESS, editedPerson);
+        String expectedMessage = String.format(EditCommand.MESSAGE_EDIT_MODULE_SUCCESS, editedModule);
 
         Model expectedModel = new ModelManager(new GradPad(model.getGradPad()), new UserPrefs());
-        expectedModel.setModule(model.getFilteredModuleList().get(0), editedPerson);
+        expectedModel.setModule(model.getFilteredModuleList().get(0), editedModule);
 
         assertCommandSuccess(editCommand, model, expectedMessage, expectedModel);
     }
 
     @Test
     public void execute_someFieldsSpecifiedUnfilteredList_success() {
-        Index indexLastPerson = Index.fromOneBased(model.getFilteredModuleList().size());
-        Person lastPerson = model.getFilteredModuleList().get(indexLastPerson.getZeroBased());
+        Index indexLastModule = Index.fromOneBased(model.getFilteredModuleList().size());
+        Module lastModule = model.getFilteredModuleList().get(indexLastModule.getZeroBased());
 
-        PersonBuilder personInList = new PersonBuilder(lastPerson);
-        Person editedPerson = personInList.withName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB)
-                .withTags(VALID_TAG_HUSBAND).build();
+        ModuleBuilder moduleInList = new ModuleBuilder(lastModule);
+        Module editedModule = moduleInList.withCode(VALID_CODE_CS3216).withModularCredits(VALID_CREDITS_CS3216)
+                .withTags(VALID_TAG_CORE).build();
```

Comment:
The `lastModule` is already CS3216 right HAHA so we don't need to "edit" it with the same code and credits again

```suggestion
        Module editedModule = moduleInList
                .withTags(VALID_TAG_CORE).build();
```